packages/integration-tests/test/actions.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { Config } from '../../../common'
+import { Web3Provider } from '@ethersproject/providers'
+import { ganache } from '@eth-optimism/ovm-toolchain'
+import assert = require('assert')
+
+const {
+  Contract,
+  ContractFactory,
+  providers: { JsonRpcProvider },
+  Wallet,
+} = require('ethers');
+const fs = require('fs')
+
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
```

Comment:
I believe there is a `sleep` function defined in the `common` file already

Reply:
Oh nice. I'll use that